ue5osc/__init__.py

```diff
@@ -8,7 +8,15 @@ def __init__(self, ip: str, port: int):
     def move_forward(self, amount: float):
         self.client.send_message("/move/forward", float(amount))
 
-    # Turn right
-    # Turn left
-    # Take screenshot
-    # Reset to beginning
+    def turn_left(self, degree: float):
+        self.client.send_message("/turn/left", float(degree))
+
+    def turn_right(self, degree: float):
+        self.client.send_message("/turn/right", float(degree))
+
+    #float parameters are optional for these two functions
+    def take_screenshot(self, degree: float=0.0):
+        self.client.send_message("/take/screenshot", float(degree))

```

Comment:
Change the address to `/screenshot/<filename>` where `<filename>` is what we want to use for the screenshot filename.

It might be better to send the filename as a argument/parameter.